modules/chronograf-security-group-rules/main.tf

```diff
@@ -2,22 +2,31 @@
 # ATTACH SECURITY GROUP RULE TO ALLOW INCOMING CONNECTIONS ON CHRONGRAF'S LISTENING PORT
 # ---------------------------------------------------------------------------------------------------------------------
 
+# ----------------------------------------------------------------------------------------------------------------------
+# REQUIRE A SPECIFIC TERRAFORM VERSION OR HIGHER
+# This module has been updated with 0.12 syntax, which means it is no longer compatible with any versions below 0.12.
+# ----------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.12"
+}
+
```

Comment:
Add a comment block that differentiates the bottom from `terraform` config. Probably something like:

```
# ------------------------------------------------------------------------------
# CREATE SECURITY GROUP RULES OPTIMIZED FOR CHRONOGRAF
# ------------------------------------------------------------------------------
```